BUILDING.md

```diff
@@ -77,14 +77,18 @@ If you want to make changes to the code and would like to use a debugger/not hav
   * Add `vrserver.exe` as the process name 
   * Ensure that `Action` is set to `Attach Debugger`.  
 
-## Launch SteamVR when building through Visual Studio
-It's usually quite useful to build then automatically launch SteamVR for debugging purposes.  
+## Launch SteamVR/Overlay when building through Visual Studio
+It's usually quite useful to build then automatically launch SteamVR/Overlay for debugging purposes.  
+The projcets includes both a driver and overlay (found in `overlay/`). Running the overlay exe automatically launches SteamVR.  
+The overlay is used for utilising methods that are not found in the OpenVR Driver API, such as discovering controllers. If you do not need this functionality, you can simply launch SteamVR.
```

Comment:
When would you not need controller discovery?

Reply:
Also, unless I did something silly and was using the steam version of the overlay, it does seem like the overlay works just by launching steamvr instead of running the overlay